src/popup/warnings/list-page.tsx

```diff
@@ -29,14 +30,13 @@ const pageStyles = {
 export default function WarningsListPage(): JSX.Element {
   return (
     <Layout title='Prod Guard'>
-      <div>
-        <div css={pageStyles.title}>
-          <h2>Warnings</h2>
-          <LinkButton to='/new'>New Warning</LinkButton>
-        </div>
-
-        <WarningList />
+      <MissingPermissionsAlert />
```

Comment:
This should probably live in the `<Layout>` since it makes sense to always show it